librz/asm/arch/arm/armass64.c

```diff
@@ -734,7 +771,7 @@ static ut32 logical(ArmOp *op, bool invert, LogicalOp opc) {
 		data |= (opc & 3) << 29;
 
 		ut32 imm_orig = op->operands[2].immediate;
-		ut32 imm = decodeBitMasksWithSize (invert? ~imm_orig: imm_orig, is64bit? 64: 32);
+		ut32 imm = encodeBitMasksWithSize (invert? ~imm_orig: imm_orig, is64bit? 64: 32);
```

Comment:
why encode? it should be decode.

Reply:
Because function with renamed to be semantically more correct: https://github.com/rizinorg/rizin/pull/351/files#diff-c085597e653efdfe11e4dadc32ab1c9b4f25609f94f6c98be6475d2d5185cc14R158

Reply:
yes but that `ut32 imm` is wrong. should be called `ut32 imm_mask`

Reply:
Right, renamed to the `imm_mask`.